keyboards/boardwalk/rules.mk

```diff
@@ -8,7 +8,7 @@ BOOTLOADER = atmel-dfu
 #   change to "no" to disable the options, or define them in the Makefile in
 #   the appropriate keymap folder that will get included automatically
 #
-BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```
See https://docs.qmk.fm/#/ChangeLog/20210828?id=bootmagic-full-removal — `BOOTMAGIC_ENABLE = yes` is the way to enable Bootmagic Lite now, and the `lite` value won't be supported after the next round of breaking changes.